xunit-autolabeler-v2/ast_parser/lib/constants.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2020 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+IGNORED_METHOD_NAMES = [
+    'run_command',
+    'parse_command_line_args',
+    'main'
+]
+
+REGION_TAG_GREP_ARGS = \
+    f'grep -hr START .'.split(' ')
```

Comment:
do you need f-string here? Also, you're using list literal below. Can you just do the same? Also, can you combine them into one assignment?

Reply:
Are you saying you'd prefer strings here, or lists?

Reply:
**Update**: changed to tuples at @martinvarelaj's request.